docs/fundamentals/code-analysis/overview.md

```diff
@@ -54,36 +56,51 @@ The following rules are enabled, by default, in .NET 7.
 
 - For a list of rules that are included with each .NET SDK version, see [Analyzer releases](https://github.com/dotnet/roslyn-analyzers/blob/main/src/NetAnalyzers/Core/AnalyzerReleases.Shipped.md).
 - For a list of all the code quality rules, see [Code quality rules](quality-rules/index.md).
 
 ### Enable additional rules
 
-*Analysis mode* refers to a predefined code analysis configuration where none, some, or all rules are enabled. In the default analysis mode, only a small number of rules are [enabled as build warnings](#enabled-rules). You can change the analysis mode for your project by setting the [\<AnalysisMode>](../../core/project-sdk/msbuild-props.md#analysismode) property in the project file. The allowable values are:
-
-:::row:::
-    :::column:::
-        `None`
-    :::column-end:::
-    :::column:::
-        `Default`
-    :::column-end:::
-    :::column:::
-        `Minimum`
-    :::column-end:::
-    :::column:::
-        `Recommended`
-    :::column-end:::
-    :::column:::
-        `All`
-    :::column-end:::
-:::row-end:::
-
-Starting in .NET 6, you can omit [\<AnalysisMode>](../../core/project-sdk/msbuild-props.md#analysismode) in favor of a compound value for the \<AnalysisLevel> property. For example, the following value enables the recommended set of rules for the latest release: `<AnalysisLevel>latest-Recommended</AnalysisLevel>`. For more information, see [AnalysisLevel](../../core/project-sdk/msbuild-props.md#analysislevel).
-
-To find the default severity for each available rule and whether or not the rule is enabled in the default analysis mode, see the [full list of rules](https://github.com/dotnet/roslyn-analyzers/blob/main/src/NetAnalyzers/Core/AnalyzerReleases.Shipped.md).
+*Analysis mode* refers to a predefined code analysis configuration where none, some, or all rules are enabled. In the default analysis mode (`Default`), only a small number of rules are [enabled as build warnings](#enabled-rules). You can change the analysis mode for your project by setting the [`<AnalysisMode>`](../../core/project-sdk/msbuild-props.md#analysismode) property in the project file. The allowable values are:
+
+| Value | Description |
+| - | - |
+| `None` | No rules are enabled. |
+| `Default` | The default set of rules are enabled. These rules are listed at [Enabled rules](#enabled-rules). |
+| `Minimum` | More aggressive mode than `Default` mode. Certain suggestions that are highly recommended for build enforcement are enabled as build warnings. To see which rules this includes, inspect the *%ProgramFiles%/dotnet/sdk/\[version]/Sdks/Microsoft.NET.Sdk/analyzers/build/config/analysislevel_\[level]_minimum.editorconfig* file. |
+| `Recommended` | More aggressive mode than the `Minimum` mode, where more rules are enabled as build warnings. To see which rules this includes, inspect the *%ProgramFiles%/dotnet/sdk/\[version]/Sdks/Microsoft.NET.Sdk/analyzers/build/config/analysislevel_\[level]_recommended.editorconfig* file. |
```

Comment:
The paths are so long with no spaces in them they cause a horizontal scroll bar on the table that makes the text hard to read